docs/server/project_setup.rst

```diff
@@ -0,0 +1,175 @@
+.. _project_setup:
+
+Create a Project
+================
+
+Now that you have the server up and running, this guide will help you get
+started with Pootle setup for getting translations.
+
+
+Assumptions
+-----------
+
+- We are setting up a project which consists of PO files and you have them on
+  hand.
+- You have a POT file with the strings you want to get translated.
+- The files are using the GNU layout, more about that later.
+
+
+.. _project_setup#add-new-project:
+
+Adding a new project
+--------------------
+
+.. _project_setup#place-translation-files:
+
+Place translation files
++++++++++++++++++++++++
+
+First task is to place the translation files for your new project in a location
+where Pootle can find them. Pootle writes and reads the translation files from
+the location defined by :setting:`POOTLE_TRANSLATION_DIRECTORY`. You can
+customize the location by altering this setting. By default it is the
+:file:`translations` directory within Pootle codebase.
+
+As told above, we will be using a GNU layout for the examples. A GNU layout
```

Comment:
Perhaps we might want to pick more about layouts from http://docs.translatehouse.org/projects/pootle/en/stable-2.5.1/features/version_control.html#how-to-treat-special-directory-layouts